### PR TITLE
remove deprecated

### PR DESCRIPTION
**Pull request by gkushang**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -129,10 +145,12 @@
     "js-base64": "^2.3.2",
     "jsonfile": "^5.0.0",
     "lodash": "^4.17.11",
+    "node-emoji": "^1.10.0",
     "open": "^6.4.0"
   },
   "devDependencies": {
     "chai": "^4.2.0",
+    "chalk": "^2.4.2",
```
Comment (lneves12) [inline package.json]: Hi @gkushang.

I think should be a dependency and not devDependency

Comment (gkushang) [reply package.json]: You’re right! Let me fix it soon in next patch version. Thanks for pointing it out !

Comment (gkushang) [reply package.json]: @lneves12 published the fix with `5.0.2`

Comment (lneves12) [reply package.json]: thanks @gkushang

**Pull request merged**